src/phoenix/db/insertion/span.py

```diff
@@ -26,17 +26,14 @@ async def insert_span(
     project_name: str,
 ) -> Optional[SpanInsertionEvent]:
     dialect = SupportedSQLDialect(session.bind.dialect.name)
-    project_rowid = await session.scalar(
-        insert_on_conflict(
```

Comment:
`insert_on_conflict` is wasteful because it's almost aways the case that `select` would suffice.